test/OpeningTimes.js

```diff
@@ -151,417 +140,372 @@ describe('OpeningTimes', () => {
+    describe('returned status value', () => {
+      const openingTimesJson = getRegularWorkingWeek();
+      const openingTimes = getNewOpeningTimes(openingTimesJson, 'Europe/London');
+      const moment = getMoment('monday', 11, 30, 'Europe/London');
+
+      describe('default values', () => {
+        const status = getStatus(openingTimes, moment);
+        it('isOpen should be populated', () => {
+          expect(status.isOpen).to.not.be.undefined;
+        });
+        it('moment should be populated', () => {
+          expect(status.moment).to.not.be.undefined;
+        });
+        it('until should not be populated', () => {
+          expect(status.until).to.be.undefined;
+        });
+      });
+      describe('passed until option', () => {
+        const status = getStatusWithUntil(openingTimes, moment);
+        it('isOpen should be populated', () => {
+          expect(status.isOpen).to.not.be.undefined;
+        });
+        it('moment should be populated', () => {
+          expect(status.moment).to.not.be.undefined;
+        });
+        it('until should be populated', () => {
+          expect(status.until).to.not.be.undefined;
+        });
+      });
+    });
+
     describe('single session (9:00 - 17:30)', () => {
       const openingTimesJson = getRegularWorkingWeek();
       const openingTimes = getNewOpeningTimes(openingTimesJson, 'Europe/London');
 
-      it('should return true if time is inside opening times', () => {
+
+      describe('moment inside opening times', () => {
         const moment = getMoment('monday', 11, 30, 'Europe/London');
-        expect(openingTimes.isOpen(moment)).to.equal(true);
+        const status = getStatusWithUntil(openingTimes, moment);
+        it('isOpen should be true', () => {
+          expect(status.isOpen).to.equal(true);
+        });
+        it('until should be next closed', () => {
+          momentsShouldBeSame(status.until, getMoment('monday', 17, 30, 'Europe/London'));
+        });
       });
 
-      it('should return false if time is outside opening times', () => {
+      describe('moment outside opening times', () => {
         const moment = getMoment('monday', 18, 30, 'Europe/London');
-        expect(openingTimes.isOpen(moment)).to.equal(false);
+        const status = getStatusWithUntil(openingTimes, moment);
+        it('status.isOpen should be false', () => {
+          expect(status.isOpen).to.equal(false);
+        });
+        it('status.until should be next open', () => {
+          momentsShouldBeSame(status.until, getMoment('tuesday', 9, 0, 'Europe/London'));
+        });
       });
     });
 
     describe('split sessions (9:00 - 12:30, 13:30 - 17:30)', () => {
       const openingTimesJson = getRegularWorkingWeekWithLunchBreaks();
       const openingTimes = getNewOpeningTimes(openingTimesJson, 'Europe/London');
 
-      it('should return false if time is during lunchtime', () => {
+      describe('moment inside lunch time', () => {
```

Comment:
It would be good to have the tests be at the boundaries. It would help highlight that when the time is equal to the closing time the state is returned as `open`.